docs/portal.properties-Reference.md

```diff
@@ -170,6 +170,13 @@ By default, the initial x-axis limit for a survival plot is the time of the late
 survival.initial_x_axis_limit=
 ```
 
+## Display installation map
+This setting specifies the URL for the iframes of a given portal's installation map on the homepage and standalone page. By default, the installation map configuration is commented out with the iframes and standalone page hidden. 
+The configuration below is used for https://www.cbioportal.org/.
```

Comment:
This is good, but let's add something about how they can set up their own installation map instance. Something like:
> The source code for the installation map can be found [here](https://github.com/cbioportal/installation-map).

Reply:
I should update the repo readme and script names sometime soon. 😅